app/Models/EntryDAO.php

```diff
@@ -578,14 +578,14 @@ public function cleanOldEntries($id_feed, $options = []) { //Remember to call up
 		if (!empty($options['keep_min']) && $options['keep_min'] > 0) {
 			//Double SELECT for MySQL workaround ERROR 1093 (HY000)
 			$sql .= ' AND `lastSeen` < (SELECT `lastSeen`'
-			      . ' FROM (SELECT e2.`lastSeen` FROM `_entry` e2 WHERE e2.id_feed = :id_feed2'
-			      . ' ORDER BY e2.`lastSeen` DESC LIMIT 1 OFFSET :keep_min) last_seen2)';
+				. ' FROM (SELECT e2.`lastSeen` FROM `_entry` e2 WHERE e2.id_feed = :id_feed2'
+				. ' ORDER BY e2.`lastSeen` DESC LIMIT 1 OFFSET :keep_min) last_seen2)';
```

Comment:
Here you could use the NOWDOC notation.

Reply:
Could be, but it is only a part of a longer string construction